src/main.rs

```diff
@@ -127,7 +127,14 @@ fn run() -> Result<()> {
         .arg(
             Arg::new("impl_debug_feature")
                 .long("impl_debug_feature")
-                .help("add feature gating for block and register debug implementation")
+                .help("Add feature gating for block and register debug implementation")
+                .action(ArgAction::Set)
+                .value_name("FEATURE"),
+        )
+        .arg(
+            Arg::new("impl_defmt")
+                .long("impl-defmt")
```

Comment:
this is not consistent with the other flags, unless we also bring in #771

Reply:
do you suggest to support both `-` and `_` for all flags, or only for old?

Reply:
If we go with #771, I think it makes sense to only alias snake case for old flags.
Thus, if we do include #771 then an alias on `impl-defmt` isn't needed, but maybe it's nice to include for familiarity.

Reply:
added alias in last commit